web/src/ehr/patient/PatientFields.vue

```diff
@@ -1,51 +1,53 @@
 <template>
-  <div>
-    <label for="ssn">SSN</label>
-    <input id="ssn" :disabled="mode === 'edit'" type="text" v-model="value.ssn">
-  </div>
-
-  <div class="grid grid-cols-2 gap-4">
+  <div class="space-y-5">
     <div>
-      <label for="firstname">First name</label>
-      <input id="firstname" type="text" v-model="value.firstName">
+      <label for="ssn">SSN</label>
+      <input id="ssn" :disabled="mode === 'edit'" type="text" v-model="value.ssn">
     </div>
 
-    <div>
-      <label for="surname">Surname</label>
-      <input id="surname" type="text" v-model="value.surname">
+    <div class="grid grid-cols-2 gap-4">
+      <div>
+        <label for="firstname">First name</label>
+        <input id="firstname" type="text" v-model="value.firstName">
+      </div>
+
+      <div>
+        <label for="surname">Surname</label>
+        <input id="surname" type="text" v-model="value.surname">
+      </div>
     </div>
-  </div>
 
-  <div>
-    <label for="gender">Gender</label>
-    <div class="custom-select">
-      <select id="gender" name="" v-model="value.gender">
-        <option value="unknown" selected>Unknown</option>
-        <option value="male">Male</option>
-        <option value="female">Female</option>
-        <option value="other">Other</option>
-      </select>
+    <div>
+      <label for="gender">Gender</label>
+      <div class="custom-select">
+        <select id="gender" name="" v-model="value.gender">
+          <option value="unknown" selected>Unknown</option>
+          <option value="male">Male</option>
+          <option value="female">Female</option>
+          <option value="other">Other</option>
+        </select>
 
-      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#444">
-        <path d="M24 24H0V0h24v24z" fill="none" opacity=".87"/>
-        <path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z"/>
-      </svg>
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#444">
+          <path d="M24 24H0V0h24v24z" fill="none" opacity=".87"/>
+          <path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z"/>
+        </svg>
+      </div>
     </div>
-  </div>
 
-  <div>
-    <label for="dob">Date of birth</label>
-    <input id="dob" type="date" :max="new Date().toISOString().split('T')[0]" v-model="value.dob">
-  </div>
+    <div>
+      <label for="dob">Date of birth</label>
+      <input id="dob" type="date" :max="new Date().toISOString().split('T')[0]" v-model="value.dob">
```

Comment:
poor man's RFC3339 parsing lol.

What is there's no timezone (`T`), will it still work?

Reply:
Also note that splitting on a character not in the string will result in an array of one item (which is the full string) so this will always work